ros2-library/src/test/java/us/ihmc/ros2/CommunicationTest.java

```diff
@@ -141,8 +137,7 @@ public void testSimpleIntraProcessCommunicationRealtime()
          Pair<Integer, Integer> messagesReceived = new MutablePair<>();
          try
          {
-            Domain domain = DomainFactory.getDomain();
-            RealtimeROS2Node node = new RealtimeROS2Node(domain, PeriodicNonRealtimeThreadScheduler::new, "ROS2CommunicationTest", "/us/ihmc");
+            RealtimeROS2Node node = new ROS2NodeBuilder().namespace("/us/ihmc").buildRealtime("ROS2CommunicationTest", PeriodicNonRealtimeThreadScheduler::new);
```

Comment:
`.namespace("/us/ihmc")` should be the default?

Reply:
I think it is. Just being set again here?